lnrpc/chainrpc/chainnotifier.swagger.json

```diff
@@ -0,0 +1,208 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "chainrpc/chainnotifier.proto",
+    "version": "version not set"
```

Comment:
"version not set"?

Reply:
Hmm... Seems to be the default value of the generator. Let's see if we can set some more useful value there.

Reply:
Didn't find out how to set a value other than the hard-coded default. I think it doesn't really matter as we'll embed the commit of lnd that we generate the API doc against in the doc itself. So it should always be clear what commit the documentation refers to. I'm not sure anyone uses these Swagger files outside of our API doc generator.

Reply:
sgtm, yeah was more just curious what that meant lol